server/opts.go

```diff
@@ -2075,10 +2183,18 @@ func parseAccounts(v interface{}, opts *Options, errors *[]error, warnings *[]er
 			*errors = append(*errors, &configErr{tk, msg})
 			continue
 		}
-		if err := stream.acc.AddStreamImport(ta, stream.sub, stream.pre); err != nil {
-			msg := fmt.Sprintf("Error adding stream import %q: %v", stream.sub, err)
-			*errors = append(*errors, &configErr{tk, msg})
-			continue
+		if stream.pre != "" {
+			if err := stream.acc.AddStreamImport(ta, stream.sub, stream.pre); err != nil {
+				msg := fmt.Sprintf("Error adding stream import %q: %v", stream.sub, err)
+				*errors = append(*errors, &configErr{tk, msg})
+				continue
+			}
+		} else {
```

Comment:
Can we just change 
if err := stream.acc.AddStreamImport(ta, stream.sub, stream.pre); err != nil {
To
if err := stream.acc.AddMappedStreamImport(ta, stream.sub, stream.pre+stream.sub); err != nil {
Thus allowing us to delete AddStreamImport (It does not do anything extra), thus delete duplicate code and different function names that would suggest that smth. different happens.

But not a blocker as this is maintainance only

Reply:
I wanted to reuse normal addStreamImport and change prefix to `to`..  ;)

The prefix historically could have the token separator `.` or not, so I need to do a bit of pre-processing. I kept error handling in AddStreamImport to keep tests passing that look for specific errors.

Reply:
ok